src/FactionsPro/FactionCommands.php

```diff
@@ -91,7 +100,7 @@ public function onCommand(CommandSender $sender, Command $command, $label, array
                                         $task = new FactionWar($this->plugin, $r);
                                         $handler = $this->plugin->getServer()->getScheduler()->scheduleDelayedTask($task, 20 * 60 * 2);
                                         $task->setHandler($handler);
-                                        $p->sendMessage("The war against $factionName and $sFaction has started!");
+                                        $p->sendMessage("The war against " . $factionName . " and " . $sFaction . " has started!");
```

Comment:
I think the correct syntax is to use double quotes and enclose variables in curly braces {} where there may be ambiguity. See http://php.net/manual/en/language.types.string.php the section on variable parsing.